src/templates/base/2019/contributors.html

```diff
@@ -263,6 +263,14 @@
   }
 }
 </style>
+<noscript>
+  <style nonce="{{ csp_nonce() }}">
+    .contributor-grid .contributor {
+      transform: translateY(0) !important;
+      opacity: 1 !important;
+    }
```

Comment:
Yeah that works. Generally do try to avoid `!important` but thought OK here as it was an ultimate noscript override. Happy to go with your suggestion too.

Reply:
Done.

Reply:
Nothing special about `noscript` here to necessitate `!important` IMO. The specificity should take care of it.